app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/main/home/addBirthday/AddBirthdayViewModel.kt

```diff
@@ -0,0 +1,79 @@
+package com.simply.birthdayapp.presentation.ui.screens.main.home.addBirthday
+
+import android.content.Context
+import android.net.Uri
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.simply.birthdayapp.data.entities.CreateBirthdayEntity
+import com.simply.birthdayapp.data.repositories.AddBirthdayRepository
+import com.simply.birthdayapp.presentation.extensions.uriToBase64
+import com.simply.birthdayapp.presentation.models.RelationshipEnum
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.stateIn
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Date
+import java.util.Locale
+
+class AddBirthdayViewModel(
+    private val addBirthdayRepository: AddBirthdayRepository,
+) : ViewModel() {
+
+    private val _name: MutableStateFlow<String> = MutableStateFlow("")
+    private val _imageUri: MutableStateFlow<Uri?> = MutableStateFlow(null)
+    private val _relationship: MutableStateFlow<RelationshipEnum?> = MutableStateFlow(null)
+    private val _dateTitle: MutableStateFlow<String> = MutableStateFlow("__.__.____")
+    private val _date: MutableStateFlow<Date?> = MutableStateFlow(null)
+
+    val name: StateFlow<String> = _name.asStateFlow()
```

Comment:
Write each reference after the property
For example 
private val _name: MutableStateFlow<String> = MutableStateFlow("")
val name: StateFlow<String> = _name.asStateFlow()